dens-transactions/src/DensTransactions/FakeProvider.ts

```diff
@@ -0,0 +1,333 @@
+import * as L from "lucid-cardano";
+
+import {
+  Address,
+  Assets,
+  Credential,
+  Datum,
+  DatumHash,
+  Delegation,
+  OutRef,
+  PolicyId,
+  ProtocolParameters,
+  Provider,
+  RewardAddress,
+  Script,
+  Transaction,
+  TxHash,
+  Unit,
+  UTxO,
+} from "lucid-cardano";
+3;
```

Comment:
```suggestion
```

Reply:
I'd be shocked if this were the only random `3`. 

I use CUA mode (ctrl-c/ctrl-v copy/paste) in emacs, and `ctrl-3` is the vertical window split command. You have to hit the ctrl-3 really fast w/ cua mode on or it gives you a 3 :P